Doc/glossary.rst

```diff
@@ -13,9 +13,16 @@ Glossary
       examples which can be executed interactively in the interpreter.
 
    ``...``
-      The default Python prompt of the interactive shell when entering code for
-      an indented code block or within a pair of matching left and right
-      delimiters (parentheses, square brackets or curly braces).
+      Can refer to:
+
+      * The default Python prompt of the interactive shell when entering code for
+        an indented code block or within a pair of matching left and right
+        delimiters (parentheses, square brackets or curly braces).
+
+      * A object of the :class:`Ellipsis` class.
+
+      * A representation of recursive items in self-referencing containers.
```

Comment:
As discussed in the BPO, lines 23 and 24 should be omitted.  The actual recursive repr is isn't guaranteed, has changed over time from `...` to `[...]` and my change yet again to `<...>` to fully distinguish it from actual Ellipsis objects.